main.go

```diff
@@ -27,7 +27,7 @@ func main() {
 	googleGroups := StringArray{}
 	redisSentinelConnectionURLs := StringArray{}
 
-	config := flagSet.String("config", "", "path to config file")
+	config := flagSet.String("config", "", "path to config file (Note when using config file, options are plural and s/-/_/: email-domain becomes email_domains and is a list)")
```

Comment:
I'm not sure about putting this note in the help string, WDYT @syscll?

I think it would probably be better to put in the configuration section of the documentation.

Also, I think `s/-/_/` could be a little too terse, can we a bit more verbose about that? (Just thinking not everyone will know sed syntax)

Reply:
That's understandable.  I wanted to clarify that the error message I saw when using the config file with singular parameters was wrong and try to indicate why.

This could all be simplified by not requiring plural parameters there, or allowing both singular and plural in both places.  Is that an option?

Reply:
I agree this would be tidier being in the documentation rather than the help message. Does it open the question that all the parameters should match the config names one to one?

Reply:
There are quite a few options that suddenly become plural. I'd guess this is down to whether the option command line flag can be specified multiple times...